### PR TITLE
Gitcoin round 15 blog post

### PR DESCRIPTION
**Pull request by wonderwomancode**
It's all in the title

### CODE REVIEW

File: unlock-protocol-com/blog/gitcoin-round-15.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+[Bloom Network](https://bloomnetwork.org/) continues to be the most well 
+supported project in both climate change and in the entire ecosystem. So far 
+they've raised more than $81k through Gitcoin Grants and we're so glad to see the 
+progress they've made with that funding. Some other projects with great potential 
+for climate impact are [Green Academy](https://green-academy.webflow.io/), 
+[Plants & Pillas](https://www.plantsandpillars3.com/), 
+[dMeter](https://dmeter.super.site/) and [Foresta](https://foresta.live/).
+
+## DeSci
+Some impressive projects leveraging Unlock Protocol for decentrailized science,
```
Comment (julien51) [inline unlock-protocol-com/blog/gitcoin-round-15.md:36]:
```suggestion

Some impressive projects leveraging Unlock Protocol for decentrailized science, 
```

File: unlock-protocol-com/blog/gitcoin-round-15.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## Climate Change
+We're sure that launching Unlock Protocol on the [Celo](https://celo.org/) 
+network is why we saw many more projects this round focused on climate change. 
+[Bloom Network](https://bloomnetwork.org/) continues to be the most well 
+supported project in both climate change and in the entire ecosystem. So far 
+they've raised more than $81k through Gitcoin Grants and we're so glad to see the 
+progress they've made with that funding. Some other projects with great potential 
+for climate impact are [Green Academy](https://green-academy.webflow.io/), 
+[Plants & Pillas](https://www.plantsandpillars3.com/),
```
Comment (ccarfi) [inline unlock-protocol-com/blog/gitcoin-round-15.md:32]: typo: should be "Plants & Pillars" (the "r" is missing)

File: unlock-protocol-com/blog/gitcoin-round-15.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+supported project in both climate change and in the entire ecosystem. So far 
+they've raised more than $81k through Gitcoin Grants and we're so glad to see the 
+progress they've made with that funding. Some other projects with great potential 
+for climate impact are [Green Academy](https://green-academy.webflow.io/), 
+[Plants & Pillas](https://www.plantsandpillars3.com/), 
+[dMeter](https://dmeter.super.site/) and [Foresta](https://foresta.live/).
+
+## DeSci
+
+Some impressive projects leveraging Unlock Protocol for decentrailized science,
```
Comment (ccarfi) [inline unlock-protocol-com/blog/gitcoin-round-15.md:37]: typo: decentrailized should be decentralized

File: unlock-protocol-com/blog/gitcoin-round-15.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+## DeSci
+
+Some impressive projects leveraging Unlock Protocol for decentrailized science, 
+or DeSci, tagged us in this round. 
+[Omni Analytics](https://twitter.com/omnianalytics) maintaining the top spot for 
+community support in this category, has continued to use the funding they've 
+recieved by adding more content to their "Learning Data Science on the Ethereum 
+Blockchain" course. Congratulations to 
+[Active Inference Institute](https://www.activeinference.org/) who caught 
+community attention with their plans to create a first of it's kind open source
```
Comment (ccarfi) [inline unlock-protocol-com/blog/gitcoin-round-15.md:44]: typo: it's should be its

File: unlock-protocol-com/blog/gitcoin-round-15.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+community support in this category, has continued to use the funding they've 
+recieved by adding more content to their "Learning Data Science on the Ethereum 
+Blockchain" course. Congratulations to 
+[Active Inference Institute](https://www.activeinference.org/) who caught 
+community attention with their plans to create a first of it's kind open source 
+package Active Blockference which aims to implement Active Inference simulations 
+for congnitive modeling for the blockchain space, use Active Blockreference to 
+understand behavior in ETH/Web3 ecosystems and improve educational materials 
+around use of Active Blockreference. 
+
```
Comment (ccarfi) [inline unlock-protocol-com/blog/gitcoin-round-15.md:49]: line 45 has Blockference and lines 46 and 48 have Blockreference  — are those all correct? (it's spelled differently on line 45 vs the others)

File: unlock-protocol-com/blog/gitcoin-round-15.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+## web3 Social
+
+We saw an increase in the number of web3 social projects taking advantage of 
+the power of Unlock Protocol too! [Nimi](https://nimi.io/) was a crowd favorite 
+with an dApp that allows you to aggregate your soical profiles. Other notable
```
Comment (ccarfi) [inline unlock-protocol-com/blog/gitcoin-round-15.md:54]: **an dApp** should be **a dApp**

File: unlock-protocol-com/blog/gitcoin-round-15.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+We saw an increase in the number of web3 social projects taking advantage of 
+the power of Unlock Protocol too! [Nimi](https://nimi.io/) was a crowd favorite 
+with an dApp that allows you to aggregate your soical profiles. Other notable 
+projects include [Cobogo](https://cobogo.io) for creator profiles and 
+fundraising for the creator economy, [Maven Foundation](https://maven.foundation/) 
+is continuing their work on building tools to empower social influencers and 
+[MeTag](https://www.getmetag.io) is creating a scannable QR tag which is also NFC 
+enabled to help users share their web3 wallet information and social profiles. 
+[Headline](https://viaheadline.xyz/) a decentrailized publishing platform which 
+got it's start with a [UDT Grant](https://docs.unlock-protocol.com/governance/grants-bounties-and-matchings) from Unlock Protocol jumped onto the Gitcoin
```
Comment (ccarfi) [inline unlock-protocol-com/blog/gitcoin-round-15.md:61]: typo: it's should be its

File: unlock-protocol-com/blog/gitcoin-round-15.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+[MeTag](https://www.getmetag.io) is creating a scannable QR tag which is also NFC 
+enabled to help users share their web3 wallet information and social profiles. 
+[Headline](https://viaheadline.xyz/) a decentrailized publishing platform which 
+got it's start with a [UDT Grant](https://docs.unlock-protocol.com/governance/grants-bounties-and-matchings) from Unlock Protocol jumped onto the Gitcoin 
+platform this round to continue their work.
+
+## Education & Security
+
+[VestLab](VestLab.io) is aiming to get you all the information about a crypto 
+project before you invest. While [White Hat DAO](https://whitehatdao.com) is
```
Comment (ccarfi) [inline unlock-protocol-com/blog/gitcoin-round-15.md:67]: don't need "While..." at the beginning here; please remove (ok to just start with White Hat DAO)

File: unlock-protocol-com/blog/gitcoin-round-15.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+the traditional payroll and contractor payments space like slow posting times 
+and by making it easier for workers to invest a portion of their paychecks. 
+[Starksheet](https://www.starksheet.xyz/) is hoping to tackle some the largest 
+data silos by decentralizing business intelligence with a web3 spreadsheet 
+solution making use of Unlock NFTs.
+
+## Diversity, Equity & Inclusion
+
+Last but not least there were several projects both utilizing Unlock Protocol 
+and tackling Diversity, Equity & Inclusiion (DEI) in web3. We continue to be
```
Comment (ccarfi) [inline unlock-protocol-com/blog/gitcoin-round-15.md:89]: type: Inclusiion should be Inclusion

File: unlock-protocol-com/blog/gitcoin-round-15.md [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+and by making it easier for workers to invest a portion of their paychecks. 
+[Starksheet](https://www.starksheet.xyz/) is hoping to tackle some the largest 
+data silos by decentralizing business intelligence with a web3 spreadsheet 
+solution making use of Unlock NFTs.
+
+## Diversity, Equity & Inclusion
+
+Last but not least there were several projects both utilizing Unlock Protocol 
+and tackling Diversity, Equity & Inclusiion (DEI) in web3. We continue to be 
+passionate about our support of these projects and were so enthusiastic to see
```
Comment (ccarfi) [inline unlock-protocol-com/blog/gitcoin-round-15.md:90]: typo: were should be we're

File: unlock-protocol-com/blog/gitcoin-round-15.md [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+
+Last but not least there were several projects both utilizing Unlock Protocol 
+and tackling Diversity, Equity & Inclusiion (DEI) in web3. We continue to be 
+passionate about our support of these projects and were so enthusiastic to see 
+so many in this round. Our friends over at [Blu3 DAO](https://www.blu3dao.com/) 
+continue to do such impressive work educating, preparing and sponsoring hackers 
+in the web3 space. The community seems to agree with our take on the work they 
+do with hundreds of supporters and thousands of dollars in community support they 
+were able to raise close to $60k this round with matching funds and we couldn't 
+be happier to see that happen for them!
```
Comment (ccarfi) [inline unlock-protocol-com/blog/gitcoin-round-15.md:96]: "The community seems to agree with our take on the work they 
do with hundreds of supporters and thousands of dollars in community support they were able to raise close to $60k this round with matching funds and we couldn't be happier to see that happen for them!" is a run-on. Please change to:

The community seems to agree with our take on the work they 
do, with hundreds of supporters and thousands of dollars in community support. They were able to raise close to $60k this round with matching funds and we couldn't be happier to see that happen for them!

File: unlock-protocol-com/blog/gitcoin-round-15.md [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+be happier to see that happen for them! 
+[End Domestic Violence](https://EndDomesticViolence.Art) is on a mission to 
+create a powerful healing modality for domestic violence victims using art to 
+process trauma and is building an art workshop platform on Unlock Protocol to 
+deliver it. They continue to be the only grantee on the platform addressing the 
+issue of domestic violence so we're watching closely to see how the community 
+reacts to the progess they'll make between this round and next as they get off 
+the ground. Another project supporting women's reproductive rights while also 
+bringing you an innovative music experience is 
+[Beam Per Mint](https://tell.ie/beatspermint/j6upJTcHktEE) who will soon be
```
Comment (ccarfi) [inline unlock-protocol-com/blog/gitcoin-round-15.md:105]: should "Beam Per Mint" be "Beats Per Mint" ???

**Review by ccarfi — APPROVED** [review]
thanks for making those edits!

**Pull request merged**